governance/votes/Executive vote - May 10, 2023.md

```diff
@@ -0,0 +1,156 @@
+* Increase RETH-A Stability Fee by 0.25% from 0.5% to **0.75%**.
+* Increase CRVV1ETHSTETH-A Stability Fee by 0.25% from 1.5% to **1.75%**.
+
+#### [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Changes
+
+* Increase the WSTETH-A `gap` by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Increase the WSTETH-B `gap` by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Reduce the WBTC-A `gap` by 10 million DAI from 20 million DAI to **10 million DAI**.
+* Reduce the WBTC-B `gap` by 5 million DAI from 10 million DAI to **5 million DAI**.
+* Reduce the WBTC-C `gap` by 10 million DAI from 20 million DAI to **10 million DAI**.
+
+#### [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) Changes
+
+* Increase the WSTETH-A `ttl` by 21,600 seconds from 21,600 seconds to **43,200 seconds**.
+* Increase the WSTETH-B `ttl` by 28,800 seconds from 28,800 seconds to **57,600 seconds**.
+
+### Starknet Bridge Limit Increase
+
+As per [this initial poll](https://vote.makerdao.com/polling/QmUnhQZy) and this [forum update](https://forum.makerdao.com/t/april-26th-2023-spell-starknet-bridge-limit/20589), the Starknet Bridge Limit will be increased as follows, if this executive proposal passes.
+
+* The Starknet Bridge Limit will be increased by 4 million DAI from 1 million DAI to **5 million DAI**.
+
+### Vesting Stream Setup
+
+As per [MIP106](https://mips.makerdao.com/mips/details/MIP106), the following DAI and MKR vesting streams will be created if this executive proposal passes:
+
+#### DAI Vesting Streams
+
+| Ecosystem Actor Name | DAI Amount | Start Date | End Date | Destination Address |
+|---|---|---|---|---|
+| PullUp Auditor Wallet | 3,300,000 | 2023-05-01 | 2024-05-01 | [0x868B44e8191A2574334deB8E7efA38910df941FA](https://etherscan.io/address/0x868B44e8191A2574334deB8E7efA38910df941FA) |
+| Phoenix Labs Auditor Wallet | 1,534,000 | 2023-05-01 | 2024-05-01 | [0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA](https://etherscan.io/address/0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA) |
+
+#### MKR Vesting Streams
+
+| Ecosystem Actor Name | MKR Amount | Start Date | End Date | Cliff Date | Destination Address |
+|---|---|---|---|---|---|
+| PullUp Auditor Wallet | 4,000 | 2023-05-01 | 2025-05-01 | 2023-05-01 | [0x868B44e8191A2574334deB8E7efA38910df941FA](https://etherscan.io/address/0x868B44e8191A2574334deB8E7efA38910df941FA) |
+| Phoenix Labs Auditor Wallet | 986.25 | 2023-05-01 | 2024-05-01 | 2023-05-01 | [0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA](https://etherscan.io/address/0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA) |
+
+### Cancel Vesting Streams
+
+As part of the above entry, the previous vesting streams to Phoenix Labs will be cancelled and replaced with the new vesting streams, if this executive proposal passes.
+
+* `yank` DAI stream ID 22.
```

Comment:
I would reword this, maybe say how much is cancelled? Idk. Probably don't use 'yank'

Reply:
added a description